modules/auxiliary/admin/http/cnpilot_r_cmd_exec.rb

```diff
@@ -0,0 +1,142 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Auxiliary::CNPILOT
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name' => "Cambium cnPilot r200/r201 Command Execution as 'root'",
+      'Description' => %{
+        Cambium cnPilot r200/r201 device software versions 4.2.3-R4 to
+        4.3.3-R4, contain an undocumented, backdoor 'root' shell. This shell is
+        accessible via a specific url, to any authenticated user. The module uses this
+        shell to execute arbitrary system commands as 'root'.
+      },
+      'Author' =>
+        [
+          'Karn Ganeshen <KarnGaneshen[at]gmail.com>'
+        ],
+      'References' =>
+        [
+          ['CVE', '2017-5259'],
+          ['URL', 'https://blog.rapid7.com/2017/12/19/r7-2017-25-cambium-epmp-and-cnpilot-multiple-vulnerabilities']
+        ],
+      'License' => MSF_LICENSE,
+      'DefaultOptions' => { 'VERBOSE' => true })
```

Comment:
IMO we should not be overriding this value in individual modules without a good reason.

Reply:
Probably a debug thing. I can toss it (and anywhere else this is set.

Reply:
yeah, it can be removed. This will also require a change in following modules where vprint_* is used:
```
epmp1000_get_chart_cmd_exec.rb:      vprint_line("#{res.body}")
cnpilot_r_cmd_exec.rb:        vprint_status('Command run did not return any results or invalid command. Note that cnPilot devices only have a restricted *nix command-set.')
cnpilot_r_cmd_exec.rb:      vprint_error("#{rhost}:#{rport} - Backdoor 'root' shell not found. Affected versions are - v4.2.3-R4 and newer. You can try to verify the shell at #{root_shell}")
cnpilot_r_cmd_exec.rb:      vprint_error("#{rhost}:#{rport} - This software version is not vulnerable. Affected versions are - v4.2.3-R4 and newer.")
cnpilot_r_fpt.rb:        vprint_status('File not found.')
cnpilot_r_fpt.rb:      vprint_error("#{rhost}:#{rport} - Could not read file. You can manually check by accessing #{final_url}.")
```